src/graphql/language/location.py

```diff
@@ -12,6 +12,20 @@ class SourceLocation(NamedTuple):
     line: int
     column: int
 
+    @property
+    def formatted(self):
+        return dict(line=self.line, column=self.column)
+
+    def __eq__(self, other):
+        if isinstance(other, dict):
+            return other == self.formatted
+        return super().__eq__(other)
+
+    def __ne__(self, other):
```

Comment:
Good catch to add `__ne__`. We need that to override the builtin method. But I think `return not self.__eq__(other)` should suffice here.